contracts/src/libraries/EigenDARollupUtils.sol

```diff
@@ -50,25 +51,64 @@ library EigenDARollupUtils {
+            // make sure that the adversaryThresholdPercentage is less than the given confirmationThresholdPercentage
             require(blobHeader.quorumBlobParams[i].adversaryThresholdPercentage 
-                < blobHeader.quorumBlobParams[i].quorumThresholdPercentage, 
+                < blobHeader.quorumBlobParams[i].confirmationThresholdPercentage, 
                 "EigenDARollupUtils.verifyBlob: adversaryThresholdPercentage is not valid"
             );
 
-            // make sure that the stake signed for is greater than the given quorumThresholdPercentage
-            require(uint8(blobVerificationProof.batchMetadata.batchHeader.quorumThresholdPercentages[uint8(blobVerificationProof.quorumThresholdIndexes[i])]) 
-                >= blobHeader.quorumBlobParams[i].quorumThresholdPercentage, 
-                "EigenDARollupUtils.verifyBlob: quorumThresholdPercentage is not met"
+            // make sure that the adversaryThresholdPercentage is at least the given quorumAdversaryThresholdPercentage
+            uint8 _adversaryThresholdPercentage = getQuorumAdversaryThreshold(eigenDAServiceManager, blobHeader.quorumBlobParams[i].quorumNumber);
+            if(_adversaryThresholdPercentage > 0){
+                require(blobHeader.quorumBlobParams[i].adversaryThresholdPercentage >= _adversaryThresholdPercentage, 
+                    "EigenDARollupUtils.verifyBlob: adversaryThresholdPercentage is not met"
+                );
+            }
+
+            // make sure that the stake signed for is greater than the given confirmationThresholdPercentage
+            require(uint8(blobVerificationProof.batchMetadata.batchHeader.signedStakeForQuorums[uint8(blobVerificationProof.quorumIndices[i])]) 
+                >= blobHeader.quorumBlobParams[i].confirmationThresholdPercentage, 
+                "EigenDARollupUtils.verifyBlob: confirmationThresholdPercentage is not met"
             );
 
+            // mark confirmed quorum in the bitmap
+            confirmedQuorumsBitmap = BitmapUtils.setBit(confirmedQuorumsBitmap, blobHeader.quorumBlobParams[i].quorumNumber);
+        }
+
+        // check that required quorums are a subset of the confirmed quorums
+        require(
+            BitmapUtils.isSubsetOf(
+                BitmapUtils.orderedBytesArrayToBitmap(
+                    eigenDAServiceManager.quorumNumbersRequired()
+                ),
+                confirmedQuorumsBitmap
+            ),
+            "EigenDARollupUtils.verifyBlob: confirmed quorums are not a subset of the required quorums"
+        );
+    }
+
+    /**
+     * @notice gets the adversary threshold percentage for a given quorum
+     * @param eigenDAServiceManager the contract in which the batch was confirmed 
+     * @param quorumNumber the quorum number to get the adversary threshold percentage for
+     * @dev returns 0 if the quorumNumber is not found
```

Comment:
There is no clause setting to 0 below for invalid quorumNumber. Does it implicit do so in Sol? Is it a good practice to set it explicitly?

Reply:
Yes because adversaryThresholdPercentage is defined as 0 by default with return it will return zero if a quorum is not found